contrib/clojure-package/src/dev/generator.clj

```diff
@@ -470,6 +476,53 @@
   (println "Generating symbol-api file")
   (write-to-file all-symbol-api-functions symbol-api-gen-ns "src/org/apache/clojure_mxnet/gen/symbol_api.clj"))
 
+;;;;;;; SymbolRandomAPI
+
+(defn gen-symbol-random-api-function [op-name]
```

Comment:
is it possible to share code between symbol api and symbol random api generation?

Reply:
Yes it is :) I am refactoring this now